File: backend/src/main/java/cohappy/backend/model/dto/response/GetNextChoreDTO.java
package cohappy.backend.model.dto.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class GetNextChoreDTO {
    private String choreCode;
    private String name;
    private String assignedTo;
    private LocalDate date;
    private boolean completed;
}

File: frontend/app/src/main/java/cohappy/frontend/client/dto/response/GetNextChoreDTO.kt
package cohappy.frontend.client.dto.response;

import com.squareup.moshi.JsonClass
import java.time.LocalDate

@JsonClass(generateAdapter = true)
data class GetNextChoreDTO(
    val choreCode : String,
    val name: String,
    val assignedTo: String,
    val date: LocalDate,
    val completed: Boolean) {
}
